### DISCUSSION TITLE
How do I earn badges

### DISCUSSION THREAD

**Discussion by SofiaSchneider-dev** [community/community#135441] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I see people that have badges but how do you eran them

**Comment by SergioVargasDev** [comment]
Here is a video tutorial: https://www.youtube.com/watch?v=oJWFfp3sGYA

**Comment by SergioVargasDev** [comment]
Also there is this repo: https://github.com/Thinkright20/Profile-Badges

**Comment by incri** [comment]
https://www.youtube.com/watch?v=oJWFfp3sGYA : this one is good

**Comment by deckrepairlittlerock** [comment]
there is a criteria like increasing followers and adding value to the community then you must be able to earn badges just like [deck repair ](https://deckrepairlittlerock.com/)

**Comment by ebndev** [comment]
Hi all, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
